### DISCUSSION TITLE
Merge failed: New town size exceeds the limit

### DISCUSSION THREAD

**Discussion by Aneroo** [TownyAdvanced/Towny#6419] [Q&A] [answered]
Is there any way to increase the new town size limit when merging towns with townyadmin? I tried to merge two towns but I recieved an error. Merge failed: New town size exceeds the limit (901). Is there a way to change this and where would I find it? I could find anything that would help me so I'm posting this here. Thank you in advance.

**Comment by LlmDl** [comment, marked as answer]
Your town_level section of the config usually dictates how many townblocks a town can have. When those two towns combine they exceed the limit of their future sized town.

You can force it with /ta town TOWNNAME forcemerge if you want. Or you can alter the town_level townBlockLimit.

**Reply by Aneroo** [reply]
Which order do I use for the town I want to merge into the other? Say I want town B to merge into A so there is only town A? Would I do /ta town A forcemerge B or /ta town B forcemerge A?

**Reply by LlmDl** [reply]
I think you use /ta town a forcemerge b, the wiki has a commands page.
